src/Cake.Web/App_Data/docs.xml

```diff
@@ -19,6 +19,7 @@
     <item title="Setup and teardown" file="fundamentals/setup-and-teardown.md" />
     <item title="Running targets" file="fundamentals/running-targets.md" />
     <item title="Script aliases" file="fundamentals/aliases.md" />
+    <item title="Directives" file="fundamentals/directives.md" />
```

Comment:
Maybe this should be called `Preprocessor directives` (and the file `preprocessor-directives.md`)